extensions/positron-r/src/provider.ts

```diff
@@ -293,58 +293,60 @@ function binFragments(): string[] {
 }
 
 export async function findCurrentRBinary(): Promise<string | undefined> {
-	if (os.platform() === 'win32') {
-		const registryBinary = findCurrentRBinaryFromRegistry();
-		if (registryBinary) {
-			return registryBinary;
-		}
-	}
+	// if (os.platform() === 'win32') {
+	// 	const registryBinary = findCurrentRBinaryFromRegistry();
+	// 	if (registryBinary) {
+	// 		return registryBinary;
+	// 	}
+	// }
 
 	const whichR = await which('R', { nothrow: true }) as string;
 	if (whichR) {
+		LOGGER.info(`Found R on PATH: ${whichR}.`);
 		if (os.platform() === 'win32') {
-			// rig puts {R Headquarters}/bin on the PATH, so that is probably how we got here.
-			// (The CRAN installer does NOT put R on the PATH.)
-			// In the rig scenario, `whichR` is anticipated to be a batch file that launches the
-			// current version of R ('default' in rig-speak):
-			// Example filepath: C:\Program Files\R\bin\R.bat
-			// Typical contents of this file:
-			// ::4.3.2
-			// @"C:\Program Files\R\R-4.3.2\bin\R" %*
-			// How it looks when r-devel is current:
-			// ::devel
-			// @"C:\Program Files\R\R-devel\bin\R" %*
-			// Note that this is not our preferred x64 binary, so we try to convert it.
-			if (path.extname(whichR).toLowerCase() === '.bat') {
-				const batLines = readLines(whichR);
-				const re = new RegExp(`^@"(.+R-(devel|[0-9]+[.][0-9]+[.][0-9]+).+)" %[*]$`);
-				const match = batLines.find((x: string) => re.test(x))?.match(re);
```

Comment:
I'm removing this entirely. It's rig-specific and sounds like something that rig might change / back off on (https://github.com/posit-dev/positron/pull/2031#issuecomment-1981892085). And I don't know of any situation where _this_ would be how we discovered an R installation that we would not discover through other means.